### PR TITLE
Torquebox image

### PR DESCRIPTION
**Pull request by jhadvig**
Torquebox image for running Rack base apps.

### CODE REVIEW

File: centos-torquebox/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# centos-torquebox
+
+FROM       centos
+MAINTAINER Jakub Hadvig <jhadvig@redhat.com>
+
+RUN mkdir -p /opt/torquebox && useradd torquebox -c"Torquebox system user" -M -ptorquebox
+
+RUN yum update --assumeyes && \
+	yum install --assumeyes java-1.7.0-openjdk.x86_64 wget unzip sqlite-devel libsqlite3x-devel nodejs010-nodejs zlib-devel libxslt-devel libxml2-devel && \
```
Comment (vbalazs) [inline centos-torquebox/Dockerfile]: hi, are you sure you need nodejs?

Comment (jhadvig) [inline centos-torquebox/Dockerfile]: Thanks for noticing that, I've overlooked that one

**Comment by vbalazs** [comment]
Sorry @jhadvig , it seems I was wrong. Apparently we need v8 for rake assets compilation therefore we need nodejs as a dependency.

See: https://github.com/openshift/ruby-19-centos/pull/1#discussion_r12703385
We also talked about it on the #openshift-dev channel (but I can't find the channel logs publicly)

Excuse me for my captiousness.

**Comment by jhadvig** [comment]
@vbalazs 
No worries, found out that I'm not precompiling the assets anyway (for now).
Will update it :)
Thx

**Comment by smarterclayton** [comment]
Is this updated?

**Comment by jhadvig** [comment]
@smarterclayton  No, have to update it so it will work with latest sti. Will do that and will notice you.

**Comment by jhadvig** [comment]
@smarterclayton I've updated the image. This image is not STI related, sorry for the comment above. Also added readme in case anyone wanted to try it.

File: centos-torquebox/bin/run [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export app_root_dir="${HOME}/src/${APP_ROOT:-.}"
+
+export RACK_ENV=${RACK_ENV:-"production"}
+export RAILS_ENV=${RAILS_ENV:-"${RACK_ENV}"}
+
+cd $app_root_dir
+
+torquebox deploy $app_root_dir --env=$RAILS_ENV
+
+torquebox run -b 0.0.0.0
```
Comment (smarterclayton) [inline centos-torquebox/bin/run]: use an exec here

**Comment by jhadvig** [comment]
@smarterclayton updated. Thanks for review :)

**Pull request merged**